CONTRIBUTING.md

```diff
@@ -46,11 +46,62 @@ In general, we follow the ["fork-and-pull" Git workflow](https://github.com/susa
 
 1. Fork the repository to your own Github account
 2. Clone the project to your machine
-3. Create a branch locally with a succinct but descriptive name
-4. Commit changes to the branch
-5. Following any formatting and testing guidelines specific to this repo
-6. Push changes to your fork
-7. Open a PR in our repository and follow the PR template so that we can efficiently review the changes.
+```
+git clone https://github.com/divanov11/Mumble.git
```

Comment:
I think this should mention, cloning the fork rather than the main repo?
```suggestion
git clone https://github.com/<your-github-username>/Mumble.git
```

Reply:
Oh, i forgot to fix that. Thanks